chain/chunks/src/lib.rs

```diff
@@ -795,17 +792,22 @@ impl ShardsManager {
             pool.remove_transactions(transactions)
         }
     }
+    fn pool_for_shard(&mut self, shard_id: ShardId) -> &mut TransactionPool {
+        match self.tx_pools.entry(shard_id) {
+            Vacant(_) => {
+                self.tx_pools.insert(shard_id, TransactionPool::new(self.random_seed(shard_id)));
```

Comment:
What's the point of using the entry API if you call insert of the hashmap itself? You can do `e.insert...` where e is the value in the `Vacant` variant

Reply:
This comment was not addressed